settings.py

```diff
@@ -1 +1,4 @@
 """Settings for the kytos/storehouse NApp."""
+
+# If you want to use a custom folder other than kytos workdir.
+CUSTOM_DESTINATION_FOLDER = ""
```

Comment:
please a small suggestion: Rename to `CUSTOM_DESTIONATION_PATH = "/tmp/"`

Reply:
@beraldoleal, the way it is, if it's empty, the code assumes that it's supposed to be stored under kytos working dir. For example, /var/lib/kytos/storehouse. If I leave the `CUSTOM_DESTIONATION_PATH = "/tmp/"`, we'll stumble upon the issue of losing this folder as soon as the server reboots. Let me know if you want me to change the logic. Because I assumed that by default the user didn't want to have a custom destinatination path.